internal/dag/cache.go

```diff
@@ -407,6 +410,12 @@ func (kc *KubernetesCache) secretTriggersRebuild(secret *v1.Secret) bool {
 		}
 	}
 
+	for _, s := range kc.ConfiguredSecretRefs {
```

Comment:
nit: Maybe add a comment as to why this is added?

Reply:
I added small comment and also updated the description of `secretTriggersRebuild()` to reflect this change. Were you thinking about longer description?